coral-component-colorinput/src/styles/index.styl

```
@@ -185,12 +189,17 @@ coral-colorinput-item {
       }
     }
   }
+
+  label {
+    display block;
+  }
 }
 
 ._coral-ColorInput-buttonWrapper {
   position: absolute;
   right: $colorinput-button-wrapper-offset;
   top: $colorinput-button-wrapper-offset;
+  text-align: center;
 
   // set opacity < 1 to create new stacking context (for checker image to work)
   opacity: .999;
```

Fix text wrapping within tooltip:
```suggestion
  opacity: .999;
  
  ._coral-ColorInput-preview-tooltip {
    min-width: max-content;
  }
```